frontend/src/models/workspace.js

```diff
@@ -60,6 +60,16 @@ const Workspace = {
       .catch(() => []);
     return history;
   },
+  updateChatFeedback: async function (chatId, feedback) {
+    const result = await fetch(`${API_BASE}/workspace/chat-feedback`, {
+      method: "POST",
+      headers: baseHeaders(),
+      body: JSON.stringify({ chatId, feedback }),
```

Comment:
add chatID as url param with workspace slug as well.

Then we can pre-check the workspace x user ownership and then update on backend.